### PR TITLE
Story-832- Remove deprecated and tabulated 

### PR DESCRIPTION
**Pull request by PayalKhanna**

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -17,7 +17,7 @@
         <maven.compiler.release>11</maven.compiler.release>
 
         <rosetta.dsl.version>9.7.0</rosetta.dsl.version>
-        <rosetta.common.version>0.0.0.master</rosetta.common.version>
+        <rosetta.common.version>0.0.0.cleanup_deprecated</rosetta.common.version>
```
Comment (hugohills-regnosys) [inline pom.xml:20]: branch name

File: src/main/java/com/regnosys/testing/projection/ProjectionTestExtension.java [hunk 2]
```diff
@@ -217,15 +200,14 @@ private ProjectionTestResult runProjection(Path projectExpectationFilePath,
             // No exceptions
             ExpectedAndActual<Boolean> error = new ExpectedAndActual<>(projectExpectationFilePath, expectation.isError(), false);
 
-            return new ProjectionTestResult(expectation.getInputFile(), expectation.getKeyValueFile(), expectation.getOutputFile(), keyValue, outputXml, validationFailures, validXml, error);
+            return new ProjectionTestResult(expectation.getInputFile(), expectation.getKeyValueFile(), expectation.getOutputFile(), null, outputXml, validationFailures, validXml, error);
```
Comment (hugohills-regnosys) [inline src/main/java/com/regnosys/testing/projection/ProjectionTestExtension.java:203]: Let's remove `ProjectionTestResult.keyValue` so you don't have to pass in null

File: pom.xml (hunk 1 above)
Comment (PayalKhanna) [reply pom.xml:20]: yes i will fix it once approved

File: src/main/java/com/regnosys/testing/projection/ProjectionTestExtension.java (hunk 2 above)
Comment (PayalKhanna) [reply src/main/java/com/regnosys/testing/projection/ProjectionTestExtension.java:203]: would not need it as we are not reoving the files from model as yet Minesh said not to delete them yet

**Review by hugohills-regnosys — APPROVED** [review]

**Pull request merged**